cmd/juju/romulus/listplans/list_plans.go

```diff
@@ -137,6 +137,11 @@ func (c *ListPlansCommand) Run(ctx *cmd.Context) (rErr error) {
 		}
 		output[i] = outputPlan
 	}
+
+	if len(output) == 0 && c.out.Name() == "tabular" {
```

Comment:
Note that I have to do this ugly "if format == tabular" stuff because the formatting framework doesn't give the formatting function access to stderr, which is where we should be writing this output... so I have to do this hack here.